README.md

```diff
@@ -166,543 +119,10 @@ There are two parameters:
 * a space number (it could just as easily have been a space name), and
 * a tuple.
 
```

Comment:
**Observation 2:** the line containing "`tarantool.Connect`" is essential for
starting a session. There are two parameters:

Reply:
Updated.